### DISCUSSION TITLE
Not able to reverify for GitHub education benefits

### DISCUSSION THREAD

**Discussion by RahulBirCodes** [community/community#53035] [GitHub Education]
My Github education discount expired today, but while trying to reverify I'm not able to submit the form. For some reason, I get a response that my school name is not visible and I don't have a bio in my user-profile, however my school name is in bold and very clearly visible in the image and my GitHub profile is not empty either. I've tried for a couple of days now and have had no success. How should I go ahead with this?

**Comment by adanowitz** [comment]
Same here. My name, school name, dates, even my salary. What are y'all looking for?

**Reply by RahulBirCodes** [reply]
Have you found a fix by any chance?
